### DISCUSSION TITLE
保存已读通知

### DISCUSSION THREAD

**Discussion by wangwang-code** [bs-community/blessing-skin-server#334] [Ideas]
通知只能打开看一次不怪吗

**Comment by g-plane** [comment]
这个可以考虑在以后进行优化。

**Comment by wangwang-code** [comment]
> 这个可以考虑在以后进行优化。

![G({Q(KNML)(~69I{DSNSB1E](https://user-images.githubusercontent.com/64469117/130342883-2f23b914-5f19-494a-8364-0a6960e905d9.jpg)
